frontend/src/components/ShootVersion/GShootVersionUpdate.vue

```diff
@@ -183,7 +183,7 @@ export default {
       return filter(this.items, item => !item.notNextMinor)
     },
     hasMoreSupportedVersions () {
-      return filter(this.items, item => item.notNextMinor && item.issu).length
+      return filter(this.items, item => item.notNextMinor && item.isSupported).length
```

Comment:
```suggestion
      return filter(this.items, item => item.notNextMinor && item.isSupported).length > 0
```
It should return a boolean value